### PR TITLE
Add transition to devel post

### PR DESCRIPTION
**Pull request by jwokaty**
Add post discussing Bioconductor's renaming the default branch to devel.

### CODE REVIEW

**Comment by jwokaty** [comment]
We should wait until this is reviewed and the TBD dates are updated before merging.

File: biocblog.Rproj
Comment (LiNk-NY) [inline biocblog.Rproj:13]: This file seems to only have changed due to spaces.  It is probably not worth including in the PR.

File: posts/2023-02-10-transition-to-devel/index.qmd
Comment (LiNk-NY) [inline posts/2023-02-10-transition-to-devel/index.qmd:82]: You may also want to include the tidyverse blog post: https://www.tidyverse.org/blog/2021/10/renaming-default-branch/

File: posts/2023-02-10-transition-to-devel/index.qmd
Comment (LiNk-NY) [inline posts/2023-02-10-transition-to-devel/index.qmd:36]: Describe changes needed for local git repositories first

File: posts/2023-02-10-transition-to-devel/index.qmd
Comment (LiNk-NY) [inline posts/2023-02-10-transition-to-devel/index.qmd:16]: Make a note about who is impacted (developers / maintainers)

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
Thanks for this @jwokaty !
It looks good :rocket: ! There are a couple of points to cover to make a clear distinction between local git repos and GitHub repositories for developers.

File: posts/2023-02-10-transition-to-devel/index.qmd
Comment (jwokaty) [reply posts/2023-02-10-transition-to-devel/index.qmd:36]: I've altered the text to state that if you have a GitHub repository, we recommend changing it there. When you do that, it gives you the information how to change your local repository. But if I understand your comment, you think I write how to do the local changes first?

Comment (LiNk-NY) [reply posts/2023-02-10-transition-to-devel/index.qmd:36]: That would be good enough I think. There could also be a section for the few developers (I presume) that only maintain their package(s) locally without a GitHub repo.

Comment (jwokaty) [reply posts/2023-02-10-transition-to-devel/index.qmd:36]: I am planning to add that. Thanks for your feedback!

**Comment by jwokaty** [comment]
@kozo2 @mblue9 We're ready to publish. We're renaming the default branch next week on March 8th.

**Pull request merged**
